src/Winton.Extensions.Configuration.Consul/ConsulConfigurationSource.cs

```diff
@@ -38,6 +41,8 @@ public string KeyToRemove
             set => _keyToRemove = value;
         }
 
+        public Func<KVPair, IEnumerable<KeyValuePair<string, string>>> ConvertToConfig { get; set; }
```

Comment:
This PR looks great. The only real question / suggestion that I have is what we could call this property? I think `ConvertToConfig` was OK when it was being invoked as an extension method, as then it read `kvPair.ConvertToConfig`. However, I wonder if a better name here would be one of the following:

* `MapConsulKVPairToConfig`
* `ConvertConsulKVPairToConfig`

What do you think?

Reply:
In my PR updates I went with `ConvertConsulKVPairToConfig` but please go ahead and update it again it if you would prefer a different term.